### DISCUSSION TITLE
배경음악 설정할 것인지 말 것인지에 대한 토론~!

### DISCUSSION THREAD

**Discussion by happy-baechuchu** [MetaAir2023/WebRPG#123] [General]
크롬 기본 설정 상 오디오 자동재생이 불가능합니다.

참고 :https://dev.classmethod.jp/articles/i-tried-the-howler-js-library/

배경음악 설정에 모두가 동의하신다면 제가 내일 강사님 컴퓨터 크롬 환경설정 변경해놓겠습니다~!

**Comment by aicompose** [comment]
실행 시에 오류 발생이 일어나지 않는다면 넣어도 좋을 것 같습니다.

**Comment by madirony** [comment]
BGM있으면 좋을 거 같아요~~!! 오류 발생이 안난다면 .. 강화 성공/실패 효과음도 ...?

**Reply by happy-baechuchu** [reply]
강화 성공 / 실패 효과음 찾아 볼게용^^!!!
